### PR TITLE
Fix compatibility with anycable gem v0.6.1 by explicit require of its config

### PR DESCRIPTION
**Pull request by Envek**
anycable-rails is stopped working after anycable 0.6.1 release (especially after https://github.com/anycable/anycable/commit/c3d6e64ae920fc26dc0545329bc8bb5fb1bddaee)

~This is naive try to place config loading somewhere to initializers without full understanding about what every initializer does.~

Fixes #66

### CODE REVIEW

File: lib/anycable/rails.rb [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 require "anycable"
 require "anycable/rails/version"
-require "anycable/rails/config"
```
Comment (palkan) [inline lib/anycable/rails.rb]: What if leave this `require`?

We load `anycable-rails `only in the context of a Rails app, so we can sure that at the time of the `anycable/config` load the Rails app is present.

Comment (Envek) [reply lib/anycable/rails.rb]: Ok. Done. Now we don't need to mess with initializers

**Comment by Envek** [comment]
Now it is same as #68

**Comment by palkan** [comment]
Closed by #68

**Pull request closed without merging**